headwindsim/SU95X
# Fix light emissive texture

## Summary of Changes

PR mainly to track the process of fixing the emissive texture of all the lights. As of right now, emissive seems to running but not properly. Color may chaneg when the light is off and light txture appear flickering when light is off.

## Screenshots (if necessary)


Beacon bottom on
![Screenshot (1681)](https://user-images.githubusercontent.com/19798208/218259231-9663795c-8f64-4df1-a3ae-e3d5cb9a3f9d.png)

Beacon bottom off
![Screenshot (1682)](https://user-images.githubusercontent.com/19798208/218259240-e543e49c-c42c-471b-adf4-90fba8c8d61c.png)

Beacon top on
![Screenshot (1678)](https://user-images.githubusercontent.com/19798208/218259343-ea2bf215-8251-45e9-a947-7ac71bfd6255.png)

Beacon top off
![Screenshot (1677)](https://user-images.githubusercontent.com/19798208/218259359-a3c7143c-6cbb-43b0-a02a-eee820999e0d.png)

Landing/Taxi lights on
![Screenshot (1663)](https://user-images.githubusercontent.com/19798208/218259403-436fedd0-ce34-4d9a-bf9c-399c072f0daf.png)

Landing/Taxi lights off
![Screenshot (1664)](https://user-images.githubusercontent.com/19798208/218259434-767bccda-87c0-4f03-b307-bb8ca03ee3dd.png)

## References



## Additional context



Discord username (if different from GitHub):

## Testing instructions



## How to download the PR for Testing

Every new commit to this PR will cause a new Superjet 100 artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **HEADWIND-SU95X** download link at the bottom of the page
